changelog_entry.yaml

```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Update SNAP values to FY2025.
```

Comment:
```suggestion
    - Update SNAP values for 2025.
```

Reply:
Fine as is but FY2025 is more accurate, since it starts in October 2024.